Obstacle_publisher:
  ros__parameters:
    # obstacle data
    obstacle_x: 2.0
    obstacle_y: 8.0
    obstacle_radius: 1.0
    obstacle_id: 1